### ISSUE TITLE
AlpineLinux image may be corrupted

### ISSUE THREAD

**Issue by nopnop2002** [LuckfoxTECH/luckfox-pico#107]
The AlpineLinux image is published here.

https://wiki.luckfox.com/Luckfox-Pico/Luckfox-Pico-Alpine-Linux-1

I downloaded it, wrote it to the SD card, and started mini-a.
The image may be corrupted.

![alpine-linux](https://github.com/LuckfoxTECH/luckfox-pico/assets/6020549/8b0f00aa-c75b-4361-83d2-d3f749c2a154)

**Comment by nopnop2002** [comment]
thank you for your reply.

I understand that if I write this firmware to Mini-B, I can use USB-WiFi, but is that correct?

**Comment by vjoco** [comment]
> thank you for your reply.
> 
> If I write this firmware to Mini-B, can I use USB-WiFi?

yes, but it's not "out of the box". you have to enable usb host support, and add the kernel drivers and firmware for your adapter. Which wifi adapter you you have? (lsusb gives vid and pid)

**Comment by nopnop2002** [comment]
I studied alpinelinux a little bit.

Perhaps I also need to change the SSID and password of the WiFi router to suit my environment?

I have almost everything available as a USB dongle.

**Comment by peterwillcn** [comment]
![Screenshot 2024-05-22 at 08 49 57](https://github.com/LuckfoxTECH/luckfox-pico/assets/323794/cc720fa9-ec27-451f-8349-b7181e79e21a)
